ios/Podfile.lock

```diff
@@ -190,11 +190,11 @@ PODS:
   - nanopb/encode (2.30908.0)
   - OpenSSL-Universal (1.1.1100)
   - PanModal (1.2.7)
-  - Permission-Camera (3.0.5):
```

Comment:
my ledger was locked when i clicked "review tx", it is now unlocked but the unlock sheet still pops up every time